docs/Buzzer.md

```diff
@@ -44,6 +44,38 @@ Sequences available in Cleanflight v1.9 and above are :
     15   ARMED                 0, 245, 10, 5	Board is armed (after pause ; repeats until board is disarmed or throttle is increased)
 
 
+### CLI " beeper "  command
+
+The beeper can be enabled/disabled with the **beeper** CLI command.
+
+Under CLI, all beeping conditions are listed either with *dump* or *beeper list*.
+
+- *beeper XX* and *beeper -XX* respectively enable / disable the XX beeping condition.
```

Comment:
I'd like the format "beeper xx ON" to match the syntax for enabling disabling CLI variables. is the xx / -xx format used in other places in the CLI?